.gitignore

```diff
@@ -9,4 +9,5 @@ __pycache__/
 .nginx
 .coverage
 staticfiles
-.vscode/
\ No newline at end of file
+.vscode/
+backup_*
```

Comment:
If in prod need to execute this script, we need create an script with the correct paths... So, we need add this new file inside the .gitignore, not?